deployments/AKS/README.md

```diff
@@ -0,0 +1,12 @@
+# Deploy KubeArmor on AKS
+
+## 1. Creating an AKS cluster
```

Comment:
Is it possible to add more details like the options in [EKS](https://github.com/kubearmor/KubeArmor/blob/main/deployments/EKS/README.md)?